repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,21 @@
+#########################
+# Gatling Configuration #
+#########################
+
+gatling {
+  charting {
+    indicators {
+      percentile1 = 50      # Value for the 1st percentile to track in the reports, the console summary and Graphite
+      percentile2 = 95      # Value for the 2nd percentile to track in the reports, the console summary and Graphite
+      percentile3 = 99      # Value for the 3rd percentile to track in the reports, the console summary and Graphite
+      percentile4 = 99.9    # Value for the 4th percentile to track in the reports, the console summary and Graphite
+    }
+  }
+  data {
+    writers = [console, file, graphite] # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
```

Comment:
If it's dropping all of the packets, it won't be able to establish a TCP connection. I'm guessing Gatling gives zero cares about being able to send Graphite data then if you got back a zero return code.

Establishing a TCP connection:

![create-tcp-connection](https://cloud.githubusercontent.com/assets/6766560/25450207/bd175468-2a83-11e7-925e-b75a75eca12f.jpg)

Of note, I tried connecting to InfluxDB on ports 13001 and 13002 this morning, and I monitored the network using Wireshark. In both cases, my computer sent a SYN packet to establish a connection (retrying a few times) and never got anything back.